### DISCUSSION TITLE
Pen input only please

### DISCUSSION THREAD

**Discussion by Cyprim** [keiwando/toonsquid#24] [Ideas]
I think it would be a pretty important feature to have an option that only lets the program take inputs from the Apple Pencil. I have tried this program for a lot of my work and it’s frustrating when the palm of my hand creates a brush stroke during the pipeline and having to go back to erase it is very tedious and too frequent

perhaps maybe it’s among the features but the handbook does not quite tell me anything about it, it would be really nice if anyone address this concern because it’s incredibly close to being possibly the best animation program in the iPad

**Comment by keiwando** [comment]
Yes, this will definitely be added with the 1.6 update that I'm currently working on. Thank you for the feedback!

**Comment by keiwando** [comment]
The 1.6 update is out and you can now disable finger drawing under `Settings > Gestures > Finger drawing`
